### ISSUE TITLE
bug in error construction?

### ISSUE THREAD

**Issue by Evgeny-A** [acmacalister/jetfire#68]
Is this line correct?

https://github.com/acmacalister/jetfire/blob/2a0811d5db6be2451a788b71724ad714ff334610/JFRWebSocket.m#L779

Shouldn't it be this way:

`details[NSLocalizedDescriptionKey] = detail;`

**Comment by acmacalister** [comment]
You are correct. I'm working on updating Jetfire to better match what we are doing in the Swift version, so I will roll this fix in there.
